vcs/template.py

```diff
@@ -1465,28 +1465,70 @@ def scalefont(self, scale):
             except:
                 pass
 
-    def plot(self, x, slab, gm, bg=0, min=None,
-             max=None, X=None, Y=None, **kargs):
-        """
-        This plots the template stuff on the Canvas.
-        It needs a slab and a graphic method.
+    def drawLinesAndMarkersLegend(self, canvas,
+                                  linecolors, linetypes, linewidths,
+                                  markercolors, markertypes, markersizes,
+                                  strings, bg=False, render=True):
+        """Draws a legend with line/marker/text inside a template legend box
+    Auto adjust text size to make it fit inside the box
+    Auto arrange the elements to fill the box nicely
+    :: Example ::
```

Comment:
@doutriaux1 format is incorrect here. Should be:
:Example:
    .. doctest:: template_drawLinesAndMarkers
        >>> ...
Where the ... is your code example. Any python lines that would return something to the screen when run on the console should be followed by the expected output:
>>> print("Like this")
'Like this'